### PR TITLE
qt_gui: Multiple gamepaths reading

### PR DESCRIPTION
**Pull request by notgonnaleo**
Related to issue: [https://github.com/shadps4-emu/shadPS4/issues/226](226)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -17,7 +17,7 @@ endif()
 
 project(shadPS4)
 
-option(ENABLE_QT_GUI "Enable the Qt GUI. If not selected then the emulator uses a minimal SDL-based UI instead" OFF)
+option(ENABLE_QT_GUI "Enable the Qt GUI. If not selected then the emulator uses a minimal SDL-based UI instead" ON)
```
Comment (notgonnaleo) [inline CMakeLists.txt:20]: TODO: Remove this file change from PR on next commit

File: CMakeSettings.json [hunk 2]
```diff
@@ -20,7 +20,12 @@
       "cmakeCommandArgs": "",
       "buildCommandArgs": "",
       "ctestCommandArgs": "",
-      "inheritEnvironments": [ "clang_cl_x64_x64" ]
+      "inheritEnvironments": [ "clang_cl_x64_x64" ],
+      "environments": [
```
Comment (notgonnaleo) [inline CMakeSettings.json:24]: TODO: Remove this file change from PR on next commit

File: src/qt_gui/game_install_dialog.cpp [hunk 3]
```diff
@@ -27,9 +32,55 @@ GameInstallDialog::GameInstallDialog() : m_gamesDirectory(nullptr) {
...
+    newItem->setText(path);
+    listWidget->insertItem(1, newItem);
+    */
+    // Browse button.
+    //auto browse = new QPushButton("Browse");
+
+    //connect(browse, &QPushButton::clicked, this, &GameInstallDialog::Browse);
+
+    QListWidget* listWidget = new QListWidget(this);
+    QFile file("shadPS4-gamepaths.txt");
```
Comment (georgemoralis) [inline src/qt_gui/game_install_dialog.cpp:51]: we use config class for saving settings

**Comment by gandalfthewhite19890404** [comment]
Any progress?

**Comment by georgemoralis** [comment]
closed due to inactive and incomplete implementation. if that changes we can reopen

**Pull request closed without merging**